SpatialGDK/Source/SpatialGDKEditor/Private/SnapshotGenerator/SpatialGDKEditorSnapshotGenerator.cpp

```diff
@@ -178,6 +189,7 @@ bool CreatePlaceholders(Worker_SnapshotOutputStream* OutputStream)
 	return true;
 }
 
+// This function is not in use.
```

Comment:
Why keep these then?

Reply:
Because we might use them again later for putting them in the snapshot. We def should keep since it was a lot of pluming to figure out initially.